src/routes/console/(migration-wizard)/index.ts

```diff
@@ -0,0 +1,23 @@
+import { createMigrationFormStore, createMigrationProviderStore } from '$lib/stores/migration';
+import { wizard } from '$lib/stores/wizard';
+import { requestedMigration } from '$routes/store';
+import { get, writable } from 'svelte/store';
+import Wizard from './wizard.svelte';
+
+export const formData = createMigrationFormStore();
+
+export function openMigrationWizard() {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    wizard.start(Wizard as any);
```

Comment:
Why do we need the `any`?

Reply:
Sometimes TS gets weird with Svelte components, but seems to not be causing any issues now. Will remove!